pilot.go

```diff
@@ -85,7 +86,27 @@ func (c *chanController) OpenChannel(target *btcec.PublicKey,
 
 	// With the connection established, we'll now establish our connection
 	// to the target peer, waiting for the first update before we exit.
-	feePerVSize, err := c.server.cc.feeEstimator.EstimateFeePerVSize(3)
+
+	// If `Autopilot.OpenChanFeeRate` has been specified, we use that.
+	// If not, we check `Autopilot.ConfTarget` next.
+	// Finally if neither of those is configured, we default to a 3 block target.
+	calcFeeRate := func(feeRate lnwire.MilliSatoshi, confTarget uint32) (lnwallet.SatPerVByte, error) {
+		// The zero values for integer and floats is 0. nil is not a valid integer or float value.
+		switch feeRate {
```

Comment:
You could write this part shorter and with less indentation with chained "if"s instead.

```go
if feeRate != 0 {
...
} else if confTarget != 0 {
...
} else {
...
}
```